graphein/protein/features/nodes/dssp.py

```diff
@@ -68,6 +68,7 @@ def add_dssp_features(G: nx.Graph) -> nx.Graph:
     return G
 
 
+# TODO: decide whether this needs to be deleted or not.
```

Comment:
This can go too. We have proper `DSSP` support in the `dssp` branch from @cch1999